components/HomeScreen/Title/Title.utils.js

```diff
@@ -1,20 +1,48 @@
 /**
- * @callback setGlobeOrientation
- * @param {number}
+ * rotates the globe
+ * @callback rotateGlobe
  */
 
 /**
  * yieldRotateGlobe.
  *
- * @param {number} globeOrientation
- * @param {function} setGlobeOrientation
+ * @param {object} args
+ * @param {number} args.globeOrientation
+ * @param {function} args.setGlobeOrientation
+ * @returns {rotateGlobe}
  */
-export function yieldRotateGlobe(globeOrientation, setGlobeOrientation) {
+export function yieldRotateGlobe(args) {
```

Comment:
See this on discussion of this function's name https://github.com/thirsty-wing/earth-app/pull/2#discussion_r1297770283